File: src/main/java/br/com/processo/service/ProcessoService.java
package br.com.processo.service;

import br.com.processo.domain.Processo;
import br.com.processo.service.exception.RegrasException;

public interface ProcessoService {
	Processo buscarProcesso(Integer numProcesso);
	
	Processo cadastrarProcesso(Processo processo) throws RegrasException;
}

File: src/main/webapp/js/service/processo.service.js
angular.module('processoApp')
       .service('processoService', processoService);
	   
	processoService.$inject = ['$http'];
	function processoService($http){
		this.getProcessoByNumeroUnico = function(numeroUnico){
			return $http.get("/app-processo/rest/processo/"+numeroUnico);
//			return $http.get("/app-processo/rest/processo", { params: {numprocesso: numeroUnico}});
		};
	};